OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -156,6 +156,7 @@
     <Compile Include="Primitives\MergedStream.cs" />
     <Compile Include="Primitives\SpatiallyPartitioned.cs" />
     <Compile Include="Primitives\ConcurrentCache.cs" />
+    <Compile Include="SelectableExts.cs" />
```

Comment:
I know you suggested this, but as far as I've noticed, these `Traits` folders usually contain only traits, right? Stuff you can attach to actors through the rule YAML files. That's why I opted for the `Game` folder. Did I understand this incorrectly about only traits being in the `Traits` folder?